openmc/deplete/cram.py

```diff
@@ -40,9 +42,15 @@ def deplete(chain, x, rates, dt, matrix_func=None):
         Updated atom number vectors for each material
     """
 
+    if not hasattr(chain, "fission_yields"):
+        fission_yields = repeat(chain.get_thermal_fission_yields())
+    else:
+        fission_yields = chain.fission_yields
```

Comment:
No because the heretofore undocumented `Chain.fission_yields` is a list of the weighted yields returned by

https://github.com/openmc-dev/openmc/blob/d10b83e06454b573e19e77375e46cb2a2040b423/openmc/deplete/operator.py#L592-L593

https://github.com/openmc-dev/openmc/blob/d10b83e06454b573e19e77375e46cb2a2040b423/openmc/deplete/operator.py#L608-L609

When it reaches `timed_deplete`, it _should_ have a length equal to the number of materials to be depleted by the operator.

I will add the `fission_yields` as a private attribute, with some validation on the setter for `fission_yields`